packages/shared/src/broker.ts

```diff
@@ -77,9 +78,12 @@ const requestValidators = (network: ChainflipNetwork) => ({
           }),
         )
         .optional(),
+      z.number().optional(),
     ])
-    .transform(([a, b, c, d, e]) =>
-      [a, b, c, d, transformObjToSnakeCase(e)].filter(isNotNullish),
+    .transform(([a, b, c, d, e, f]) =>
+      [a, b, c, d, transformObjToSnakeCase(e), f].map((value) =>
+        isUndefined(value) ? null : value,
```

Comment:
`undefined` in an array gets converted to `null` in JSON:

```ts
console.log(JSON.stringify([undefined]));
// => '[null]'
```

```suggestion
        value,
```

Reply:
Gut point

Reply:
I just removed that logic